build-aux/m4/bitcoin_qt.m4

```diff
@@ -76,6 +76,13 @@ AC_DEFUN([BITCOIN_QT_INIT],[
     [use_dbus=$withval],
     [use_dbus=auto])
 
+  dnl Android doesn't support D-Bus and certainly doesn't use it for notifications
```

Comment:
Please update the help message above for `--with-qtdbus`'s default

Reply:
I was considering that but I feel that the configuration should be as independent as possible from the depends build or whatever libs we decide to use. For example a user might point to prebuilt Qt for Android libs (with QtDbus) and if that breaks the build he is going to blame the `configure` script.

There's definitely no use for D-Bus on Android so I think it okay to disable even if the lib is found.

Reply:
Maybe some Android fork might add D-Bus, though? At least allow --with-dbus to override...